notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>NIT: geodesic gamma</p><p> </p><p>(not gamma(t))</p><p> </p><p>Refrain from talking about the length of the geodesic (the intuition described in this cell is correct as long as we have defined the metric)</p><p> </p><p>Typo: geodeic -&gt; geodesic</p><p> </p><p>You can also say that the Log is the inverse map of the exponential.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
<p>done w/ typo stuff, but again am wondering if it would be okay to replace length --&gt; "length" (adding quotations) and adding a sentence at the end that says that we can't specify what length is yet. I am adding it to the current version, and when you review it again maybe you can decide what you think, and i can change it later if you still think it is a bad idea.</p>